kuma/feeder/management/commands/update_feeds.py

```diff
@@ -114,7 +114,7 @@ def fetch_feed(self, feed):
         if feed.etag is None:
             feed.etag = ''
         if feed.last_modified is None:
-            feed.last_modified = datetime.datetime(1975, 1, 10)
```

Comment:
Same, please just keep on using the datetime module directly instead of going through the timezone module.